atomics/T1030/T1030.yaml

```diff
@@ -31,3 +31,40 @@ atomic_tests:
     cleanup_command: |
       if [ -f #{folder_path}/safe_to_delete ]; then rm -rf #{folder_path}; fi;
     name: sh
+
+- name: Network-Based Data Transfer in Small Chunks
+  description: "Simulate transferring data over a network in small chunks to evade detection."
+  supported_platforms:
+    - "windows"
+  auto_generated_guid: 9dc4dbee-898c-4237-8484-9f28574b8538
+  input_arguments:
+    source_file_path:
+      description: "Path to the source file to transfer."
+      type: "string"
```

Comment:
could you set the type to "path" in this case

Reply:
Updated